src/screens/LoginPage/LoginPage.tsx

```diff
@@ -223,6 +232,72 @@ function loginPage(): JSX.Element {
               <PalisadoesLogo className={styles.palisadoes_logo} />
               <p className="text-center">{t('fromPalisadoes')}</p>
             </div>
+
+            <div className={styles.socialIcons}>
+              <a
+                href="https://www.facebook.com/palisadoesproject"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaFacebook
+                  className={`${styles.socialIcon} ${styles.facebookIcon}`}
+                />
+              </a>
+              <a
+                href="https://twitter.com/palisadoesorg?lang=en"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaXTwitter
+                  className={`${styles.socialIcon} ${styles.twitterIcon}`}
+                />
+              </a>
+              <a
+                href="https://www.linkedin.com/company/palisadoes/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaLinkedin
+                  className={`${styles.socialIcon} ${styles.linkedinIcon}`}
+                />
+              </a>
+              <a
+                href="https://github.com/PalisadoesFoundation"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaGithub
+                  className={`${styles.socialIcon} ${styles.githubIcon}`}
+                />
+              </a>
+              <a
+                href="https://www.youtube.com/@PalisadoesOrganization"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaYoutube
+                  className={`${styles.socialIcon} ${styles.youtubeIcon}`}
+                />
+              </a>
+              <a
+                href="https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-29oltereu-qJ931LcKxswuCAy29iA9WA"
```

Comment:
This URL is only valid for 30 days. It changes constantly. Use:
- https://www.palisadoes.org/slack